docs/windows.md

```diff
@@ -59,7 +59,43 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (containerd based runtimes)
+
+First install antrea (v0.13.0+ is required for containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will setup antrea as a Windows service.
+
+```bash
+$TAG = "v0.13.0"
+
+curl.exe -LO "https://raw.githubusercontent.com/vmware-tanzu/antrea/${TAG}/hack/windows/Helper.psm1"
+curl.exe -LO "https://github.com/vmware-tanzu/antrea/releases/download/${TAG}/antrea-agent-windows-x86_64.exe"
+
+mv antrea-agent-windows-x86_64.exe c:/k/antrea/bin/antrea-agent.exe
+Import-Module ./helper.psm1
+Install-AntreaAgent -KubernetesVersion "v1.20.2" -KubernetesHome "c:/k" -KubeConfig "C:/etc/kubernetes/kubelet.conf" -AntreaVersion "${TAG}" -AntreaHome "c:/k/antrea"
+New-KubeProxyServiceInterface
+
+nssm install kube-proxy "c:/k/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=C:/etc/kubernetes/kubelet.conf --log-dir=c:/var/log/kube-proxy --logtostderr=false --alsologtostderr"
+nssm install antrea-agent "c:/k/antrea/bin/antrea-agent.exe" "--config=c:/k/antrea/etc/antrea-agent.conf --logtostderr=false --log_dir=c:/k/antrea/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4" 
+
+nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd 
+nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
+
+start-service kube-proxy
+start-service antrea-agent
+```
+
+### Installation via wins (docker based runtimes)
+
+Installing antrea as a wins service gives you a lot of flexibilities to manage it as a Pod, but currently this only works
+with docker due to a bug in the way containerd attaches host networks for Windows Pods. In any case, if you are using
```

Comment:
what does "attaches" mean here? maybe it should be "in the way containerd handles host networking for Windows Pods", with a link to https://github.com/containerd/containerd/issues/4856?

Reply:
Thanks for your suggestion. Updated.